.github/workflows/chromatic.yml

```diff
@@ -0,0 +1,41 @@
+# .github/workflows/chromatic.yml
+
+name: 'Chromatic'
+
+on: push
+
+jobs:
+  chromatic-deployment:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # 👈 Required to retrieve git history
+
+      - name: Setup Node
+        uses: actions/setup-node@v1
+        with:
+          node-version: "12"
+          registry-url: "https://npm.pkg.github.com"
+
+      - name: Cache dependencies
+        uses: actions/cache@v2
+        with:
+          path: ~/.npm
+          key: npm-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            restore-keys: npm-
+
+      - name: Install dependencies
+        if: steps.cache.outputs.cache-hit != 'true'
```

Comment:
is this id steps cache ever set as true? If you see this example, you'll notice the id of the step is used for this logic https://github.com/WPMedia/engine-theme-sdk/blob/arc-themes-release-version-1.19/.github/workflows/pr-test.yml#L25

Reply:
Good call! I've that on line 23 now.